docker/cassandra-sidecar/entry-point

```diff
@@ -1,8 +1,14 @@
 #!/bin/bash -ue
 
+params=""
+
 if [ "${1-unset}" == "backup" ] || [ "${1-unset}" == "restore" ]; then
     cassandra-backup-restore $@
     exit $?
 else
-    cassandra-sidecar $@
+    if [[ -e /var/lib/cassandra/tls/jmx-params ]]; then
```

Comment:
yes, we need this as the sidecar containers won't know the password it should use with the certificate. This file includes the password so that sidecar can load it into the environment.